api/services/ocean/gke/paths/clusterFetchElastilog.yaml

```diff
@@ -2,8 +2,10 @@ summary: Ocean for GKE
 get:
   summary: "Fetch Elastilog"
   description: >
-    Fetch an Ocean Cluster's log
-  operationId: "oceanGkeFetchLog"
```

Comment:
@vlad666spot why changing the operation id?

Reply:
fixed